python/cuspatial/cuspatial/core/binpreds/feature_contains_properly.py

```diff
@@ -0,0 +1,240 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.
+
+from typing import TypeVar
+
+import cupy as cp
+
+import cudf
+
+from cuspatial.core.binpreds.binpred_interface import (
+    BinPred,
+    ContainsOpResult,
+    ImpossiblePredicate,
+    NotImplementedPredicate,
+    PreprocessorResult,
+)
+from cuspatial.core.binpreds.contains import contains_properly
+from cuspatial.core.binpreds.feature_contains import (
+    ComplexGeometryPredicate,
+    ContainsPredicateBase,
+)
+from cuspatial.utils.binpred_utils import (
+    LineString,
+    MultiPoint,
+    Point,
+    Polygon,
+    _false_series,
+    _is_complex,
+)
+from cuspatial.utils.column_utils import (
+    contains_only_polygons,
+    has_multipolygons,
+)
+
+GeoSeries = TypeVar("GeoSeries")
+
+
+class ContainsProperlyPredicate(
+    ContainsPredicateBase, ComplexGeometryPredicate
```

Comment:
`ContainsPredicateBase` already inherits `ComplexGeometryPredicate`. We should be able to remove this inheritance?

Reply:
Yes, I'll address this.

Reply:
I cleaned up all the inheritance.